invenio_circulation/records/jsonresolver/item.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2018 CERN.
+# Copyright (C) 2018 RERO.
+#
+# Invenio-Circulation is free software; you can redistribute it and/or modify
+# it under the terms of the MIT License; see LICENSE file for more details.
+
+"""Circulation Item JSON Resolver module."""
+
+import jsonresolver
+from flask import current_app
+
+
+@jsonresolver.route("/api/resolver/circulation/items/<pid_value>", host="127.0.0.1:5000")
```

Comment:
Here is when we start to have issue unfortunately.
`JSONSCHEMA_HOST` will be configured in `invenio-app-ils` or upper flavors, so hardcoding this will just fail as soon as it changes.

I guess we need to expose this `host` as constant, use it here and make it available for who is using this module.
WDYT @topless @zzacharo ?